tokio/src/runtime/blocking/pool.rs

```diff
@@ -51,6 +52,7 @@ struct Shared {
     num_notify: u32,
     shutdown: bool,
     shutdown_tx: Option<shutdown::Sender>,
+    worker_threads: Vec<thread::JoinHandle<()>>,
```

Comment:
This will grow the vec in an unbounded way until the runtime is shutdown. When a thread terminates due to a timeout, the associated join handle should be removed. A scan is probably fine in this case as it means the thread is idle and there *shouldn't* be too many entries. Another option is to use the `Slab` structure (`slab` crate, already a dependency) to store join handles.

Reply:
Indeed, I haven't think about it. Now fixed.